sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -846,6 +854,8 @@ def expand(self, pcoll):
       return pcoll | 'WriteToBigQuery' >> ParDo(bigquery_write_fn)
     else:
       from apache_beam.io.gcp import bigquery_file_loads
+      assert not standard_options.streaming, (
```

Comment:
I'd suggest to `raise NotImplementedError`, since this is an actionable error for the user related to how a user uses Beam. We usually use assertions to guard against errors in development, so that a failed assertion indicates an error in the library code, not with how the library is used.